Tests/SonarQube.MSBuild.Tasks.IntegrationTests/TargetsTests/SetStyleCopPropertiesTargetTests.cs

```diff
@@ -0,0 +1,182 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="SonarIntegrationTargetsTests.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Microsoft.Build.Construction;
+using Microsoft.Build.Execution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.Linq;
+using TestUtilities;
+
+namespace SonarQube.MSBuild.Tasks.IntegrationTests.TargetsTests
+{
+    [TestClass]
+    public class SetStyleCopPropertiesTargetTests
+    {
+        public TestContext TestContext { get; set; }
+
+        #region Tests
+
+        [TestMethod]
+        [Description("Checks the target is not executed if RunSonarQubeAnalysis is not set")]
+        public void StyleCop_RunSonarQubeAnalysisNotSet()
+        {
+            // Arrange
+            string rootInputFolder = TestUtils.CreateTestSpecificFolder(this.TestContext, "Inputs");
+
+            WellKnownProjectProperties properties = new WellKnownProjectProperties();
+            properties.RunSonarQubeAnalysis = "false";
```

Comment:
Ok. @dbolkensteyn 's last change to the targets file removed the RunSonarQubeAnalysis variable. I'll update the targets file and fix up the tests.